src/main/java/se/sundsvall/seabloader/scheduler/notifier/NotifierSchedulerService.java

```diff
@@ -0,0 +1,118 @@
+package se.sundsvall.seabloader.scheduler.notifier;
+
+import static java.lang.String.format;
+import static org.springframework.util.CollectionUtils.isEmpty;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.EXPORT_FAILED;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.IMPORT_FAILED;
+
+import java.util.EnumMap;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import generated.se.sundsvall.messaging.Email;
+import generated.se.sundsvall.messaging.EmailRequest;
+import se.sundsvall.seabloader.integration.db.InvoiceRepository;
+import se.sundsvall.seabloader.integration.db.model.enums.Status;
+import se.sundsvall.seabloader.integration.messaging.MessagingClient;
+import se.sundsvall.seabloader.scheduler.AbstractScheduler;
+
+@Service
+@ConfigurationProperties("scheduler.notifier.cron")
+public class NotifierSchedulerService extends AbstractScheduler {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(NotifierSchedulerService.class);
+	private static final String LOG_EXECUTE_STARTED = "Executing started";
+	private static final String LOG_EXECUTE_ENDED = "Executing ended";
+	private static final String NOTIFICATION_SUBJECT = "Failed records discovered in %s (%s)";
+	private static final String NOTIFICATION_BODY_INTRODUCTION = "Failed record(s) exist in %s-database! %n";
+	private static final String NOTIFICATION_BODY_ROW = "%n%-20s\t%s records";
+
+	@Value("${spring.application.name:}")
+	private String applicationName;
+
+	@Value("${spring.profiles.active:}")
+	private String applicationEnvironment;
+
+	@Value("${notification.mail.recipient.address:}")
+	private String recipientAddress;
+
+	@Value("${notification.mail.sender.address:}")
+	private String senderAddress;
+
+	@Autowired
+	private InvoiceRepository invoiceRepository;
+
+	@Autowired
+	private MessagingClient messagingClient;
+
+	@Override
+	@Scheduled(cron = "${scheduler.notifier.cron.expression:-}")
+	public void execute() {
+		LOGGER.info(LOG_EXECUTE_STARTED);
+		sendFailureNotification();
+		LOGGER.info(LOG_EXECUTE_ENDED);
+	}
+
+	/**
+	 * Sends failure notifications to configured recipient.
+	 */
+	private void sendFailureNotification() {
+
+		// Collect data
+		final var failedStatusMap = new EnumMap<Status, Long>(Status.class);
+		Stream.of(Status.values()).forEach(status -> failedStatusMap.put(status, invoiceRepository.countByStatusIn(status)));
```

Comment:
What about filtering the Status enum and only include EXPORT_FAILED and IMPORT_FAILED here as there is no need to count rows with other statuses than those? I imagine that you could gain several nanoseconds in execution time that way 😄.